### PR TITLE
Add DateTime to all Pull Request Comments

### PR DESCRIPTION
**Pull request by sbowenwilliams**
image

### CODE REVIEW

File: src/on_pull_request.ts [hunk 1]
```diff
@@ -132,3 +132,18 @@ async function slack(hook: string, msg: string) {
...
+
+function getDateTime() {
+  const date_ob = new Date();
+  const date = intTwoChars(date_ob.getDate());
+  const month = intTwoChars(date_ob.getMonth() + 1);
+  const year = date_ob.getFullYear();
+  const hours = intTwoChars(date_ob.getHours());
+  const minutes = intTwoChars(date_ob.getMinutes());
+  const seconds = intTwoChars(date_ob.getSeconds());
+  return `\n${hours}:${minutes}:${seconds} ${month}/${date}/${year}`;
```
Comment (stevenweber) [inline src/on_pull_request.ts:144]: There are some pretty decent [native ones](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#Conversion_getter), e.g.

```js
new Date().toString()
"Mon Apr 13 2020 14:22:09 GMT-0500 (Central Daylight Time)"
new Date().toISOString()
"2020-04-13T19:22:15.034Z"
```

Is there any benefit to this custom form?

Comment (sbowenwilliams) [reply src/on_pull_request.ts:144]: Absolutely not, I didn't realize those native options were available.  Thank you!

File: .gitignore [hunk 2]
```diff
@@ -99,3 +99,5 @@ __tests__/runner/*
 lib/**/*
 
 test.js
+
+*.swp
```
Comment (ebenoist) [inline .gitignore:103]: ty

File: __tests__/main.test.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import * as nock from 'nock';
+import * as mockdate from 'mockdate'
```
Comment (ebenoist) [inline __tests__/main.test.ts:2]: nice

File: src/on_pull_request.ts [hunk 4]
```diff
@@ -114,7 +114,7 @@ async function comment(octokit: github.GitHub, issue, body: string) {
     owner: issue.owner,
     repo: issue.repo,
     issue_number: issue.number,
-    body: body
+    body: body.concat(getDateTime())
```
Comment (ebenoist) [inline src/on_pull_request.ts:117]: I thought the linter complained about no trailing commas? no worries either way.

File: src/on_pull_request.ts [hunk 5]
```diff
@@ -132,3 +132,7 @@ async function slack(hook: string, msg: string) {
 function pp(obj: Record<string, any>): string {
   return JSON.stringify(obj, undefined, 2);
 }
+
+function getDateTime() {
+  return `\n ${ new Date().toString() }`
```
Comment (ebenoist) [inline src/on_pull_request.ts:137]: Any reason for the trailing spaces if we're using a newline?

**Comment by ebenoist** [comment]
lgtm, thanks!

File: src/on_pull_request.ts (hunk 4 above)
Comment (stevenweber) [reply src/on_pull_request.ts:117]: It doesn't, got a fix coming for that though.

**Pull request merged**